lib/pliny/log.rb

```diff
@@ -4,6 +4,10 @@ def log(data, &block)
       log_to_stream(stdout || $stdout, merge_log_contexts(data), &block)
     end
 
+    def log_without_context(data, &block)
```

Comment:
@brandur After trying this out, I realized that having at least the `default_context` makes sense. Otherwise it might be hard to search for log base on a single app: `app=my-pliny-app deployment=production canonical-log-line ...`